app/src/main/java/com/android/streetworkapp/ui/authentication/SignIn.kt

```diff
@@ -81,19 +99,102 @@ fun SignInScreen(navigationActions: NavigationActions, userViewModel: UserViewMo
         modifier = Modifier.fillMaxSize().testTag("loginScreenColumnContainer"),
         verticalArrangement = Arrangement.Center,
         horizontalAlignment = Alignment.CenterHorizontally) {
-          // Welcome text in the center
+          Image(
+              painter = painterResource(id = R.drawable.app_logo),
+              contentDescription = "App Logo",
+              modifier = Modifier.height(225.dp).width(225.dp).testTag("loginScreenAppLogo"))
+
+          Spacer(modifier = Modifier.height(26.dp).testTag("loginScreenFirstSpacer"))
+
           Text(
-              text = "Welcome to the Street Work'App",
-              style = TextStyle(fontSize = 24.sp),
-              modifier = Modifier.testTag("loginTitle"))
+              text = "Welcome to Street WorkApp!",
+              style =
+                  TextStyle(
+                      fontSize = 26.sp,
+                      lineHeight = 26.sp,
+                      fontWeight = FontWeight(500),
+                      color = Color(0xFF000000),
```

Comment:
(Same as the previous comment about the `ColorPalete`)

Reply:
It's also fixed here!